ibm_ace/tests/test_config.py

```diff
@@ -0,0 +1,38 @@
+# (C) Datadog, Inc. 2022-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import pytest
+
+from datadog_checks.ibm_ace import IbmAceCheck
+
+from .common import skip_windows_ci
+
+pytestmark = [skip_windows_ci, pytest.mark.usefixtures('dd_environment'), pytest.mark.unit]
+
+
+@pytest.mark.parametrize(
+    "expected_res_stats, expected_msg_flows",
+    [
+        pytest.param(False, False, id="Metric collection disabled"),
+        pytest.param(True, False, id="Resource stats collection enabled; message_flows disabled"),
+        pytest.param(False, True, id="Resource stats collection disabled; message_flows enabled"),
+    ],
+)
+def test_config(dd_run_check, aggregator, expected_res_stats, expected_msg_flows):
+    test_instance = {
+        'mq_server': 'localhost',
+        'mq_port': 11414,
+        'channel': 'DEV.ADMIN.SVRCONN',
+        'queue_manager': 'QM1',
+        'mq_user': 'admin',
+        'mq_password': 'passw0rd',
+        'resource_statistics': expected_res_stats,
+        'message_flows': expected_msg_flows,
+        'tags': ['foo:bar'],
+    }
+
+    check = IbmAceCheck('ibm_ace', {}, [test_instance])
+    dd_run_check(check)
+
+    assert check.config.resource_statistics == expected_res_stats
```

Comment:
Sorry I did not see this before, but could we also test that subscriptions are properly updated in this test? ie `ResourceStatisticsSubscription` vs `FlowMonitoringSubscription`